src/main/java/graphql/language/InterfaceTypeDefinition.java

```diff
@@ -21,7 +21,7 @@
 import static graphql.language.NodeChildrenContainer.newNodeChildrenContainer;
 
 @PublicApi
-public class InterfaceTypeDefinition extends AbstractDescribedNode<InterfaceTypeDefinition> implements ImplementingTypeDefinition<InterfaceTypeDefinition>, DirectivesContainer<InterfaceTypeDefinition>, NamedNode<InterfaceTypeDefinition>, SDLExtensionDefinition {
+public class InterfaceTypeDefinition extends AbstractDescribedNode<InterfaceTypeDefinition> implements ImplementingTypeDefinition<InterfaceTypeDefinition>, DirectivesContainer<InterfaceTypeDefinition>, NamedNode<InterfaceTypeDefinition> {
 
```

Comment:
@bbakerman tagging you as you're on the git history for this line way back in 2021

Was this a typo, and `SDLExtensionDefinition` was meant to be an implemented interface in `InterfaceTypeExtensionDefinition` instead?

At the moment, in that class we have:
```
public class InterfaceTypeExtensionDefinition extends InterfaceTypeDefinition {
```